pkg/reconciler/openshift/common/cabundle.go

```diff
@@ -75,3 +75,43 @@ func toUnstructured(v interface{}) (*unstructured.Unstructured, error) {
 	}
 	return ud, nil
 }
+
+// ApplyCABundlesForStatefulSet is a transformer that add the trustedCA volume, mount and
```

Comment:
Can we update the comment and also I do not see anything we are doing with the environment variable